packages/dd-trace/test/datastreams/encoding.spec.js

```diff
@@ -31,6 +31,19 @@ describe('encoding', () => {
       expect(decoded2).to.equal(n)
       expect(bytes2).to.length(0)
     })
+    it('encoding can be used to encode multiple numbers in a row', () => {
+      const n1 = 1680535452135000
+      const n2 = 1680535452178000
+      const encodedN1 = encodeVarint(n1)
+      const encodedN2 = encodeVarint(n2)
+      const encoded = new Uint8Array(encodedN1.length + encodedN2.length)
+      encoded.set(encodedN1)
+      encoded.set(encodedN2, encodedN2.length)
```

Comment:
Happens to work because they're currently the same length, but I think it'd be more correct to use the length of N1, not N2.

```suggestion
      encoded.set(encodedN2, encodedN1.length)
```